src/plugins/console/common/constants/copy_as.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// Default language used in the language selector modal
+export const DEFAULT_LANGUAGE = 'cURL';
+// Available languages in the language selector modal
+export const AVAILABLE_LANGUAGES = ['cURL', 'Javascript', 'Python'];
```

Comment:
I know its a bit odd but its the way the software is called https://en.wikipedia.org/wiki/CURL.. I think I'll rather leave as is since most of our users are more on the technical side

Reply:
I don't think it's a question of technical or non-technical TBH. It's just easier on the eyes 😃. 

The official man page uses `curl`:
https://curl.se/docs/manpage.html

Not bike-shedding I swear `:homer_disappearing:`

Reply:
Related, we should use `JavaScript` instead of `Javascript`, the `S` is uppercased.